BryantUniversity/BryantUniversity/Controllers/FacultyController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
+
+namespace BryantUniversity.Controllers
+{
+    public class FacultyController : Controller
+    {
+        private Context context;
+
+        public FacultyController()
+        {
+            context = new Context();
+        }
+
+        // GET: Faculty
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        public ActionResult Create()
+        {
+            var userViewModel = new UserViewModel();
+            return View(userViewModel);
+        }
+
+        [HttpPost]
+        public ActionResult Create(UserViewModel userViewModel)
+        {
+            if (ModelState.IsValid)
+            {
+                string hashedPassword = BCrypt.Net.BCrypt.HashPassword(userViewModel.Password);
+                User user = new User(userViewModel.Email, hashedPassword, userViewModel.Name);
+                var repository = new UserRepo(context);
+                repository.Insert(user);
+
+                var roleRepo = new UserRoleRepo(context);
+                UserRole userRole = new UserRole();
```

Comment:
You should just declare the variable here like: UserRole userRole; because the constructor is being called in one of the switch cases. This is kind of instantiating a new UserRole object for no reason